sketch/README.md

```diff
@@ -0,0 +1,197 @@
+
+A way to find out the baud rate of the module, is to try an edited passthrough sketch from the TTN library.
+
+```Arduino
+#define debugSerial SerialUSB
+#define espSerial Serial
+
+void setup()
+{
+	while (!debugSerial)
+		;
+
+	debugSerial.begin(9600);
+
+	delay(500);
+
+	//Possible baud rates to try: 9600, 19200, 38400, 57600. Don’t forget to change the baud rate in the serial monitor accordingly
+	espSerial.begin(9600);
+	
+	debugSerial.println(F("Check if the communication with your ESP module is working by entering the following command 'AT'. Response should be ‘OK’ "));
+	debugSerial.println(F("Check the baudrate of your ESP module by entering the following command 'AT+CIOBAUD?' "));
+}
+
+void loop()
+{
+	while (debugSerial.available())
+	{
+		espSerial.write(debugSerial.read());
+	}
+	while (espSerial.available())
+	{
+		debugSerial.write(espSerial.read());
+	}
+}
+```
+
+There are allot of ‘AT’ commands you can send to the device, to configure or check settings. A good reference guide can be found here: [AT commands guide](https://room-15.github.io/blog/2015/03/26/esp8266-at-command-reference/)
+
+A good response looks like this.
+
+<img src="resources/ATOK.png" alt="Image of a good response" width="1000" align="middle">
+
+If you are only getting back garbage values, like seen here.
+
+<img src="resources/WrongBaudrate.png" alt="Image of a garbage response" width="1000" align="middle">
+
+Some extra hardware will be needed. For our use case, a FTDI programmer was used. There are other methods to configure the ESP. Like comunicate with it through an [*arduino*](http://www.martyncurrey.com/arduino-to-esp8266-serial-commincation/), but we will only be looking at the FTDI one. 
+
+*NOTE: It is worth mentioning that if the baudrate is indeed 115200 the example used with the arduino is not going to work, as it is using the softserial library, which can only go up to 57600. 
+A work around would be to use a mega which has multiple hardware serials, which can handle the 115200 baudrate.*
+
+With the FTDI it is possible to connect the ESP directly to a pc, and issue commands.
+
+Here is how you connect the FTDI:
+ 
+<img src="resources/FDTI-to-ESP.png" alt="FTDI hookup" width="700" align="middle">
+ 
+When the ESP is connected to your pc through your FTDI, it is possible to communicate with the device. This can be done with your preferred serial terminal program, *E.G.* Putty, Termite, Minicom. To name a few. 
+It is also possible to use the arduino serial monitor. The only parameter you need to know is the serial port with which the ESP is connected and the baud rate. 
+The serial port can be found in the Arduino IDE, under the `tools section`,
```

Comment:
What section?

Reply:
![image](https://user-images.githubusercontent.com/24267554/33614257-7e644012-d9df-11e7-8675-9241bceeb9a9.png)
I mean this. would: on the menu bar under the tool tab be a better description?

Reply:
The `Tools` menu, yes